src/js/containers/MinifyCard/MinifyCard.js

```diff
@@ -37,7 +37,7 @@ class MinifyCard extends Component {
         };
 
         checkboxValueList.forEach(function(value){
-            apiValueList[value] = 'on';
+            apiValueList[value.value] = 'on';
```

Comment:
let's change the name from `value.value` to `item.value` or `checkboxitem.value`